examples/nlp/intent_detection_slot_tagging/joint_intent_slot_with_bert.py

```diff
@@ -21,6 +21,7 @@
 from transformers import BertTokenizer
```

Comment:
consider using NemoBertTokenizer - otherwise all the models from the docs won't work

Reply:
Changed to the new style. Please check it out.